src/main/java/org/sonarlint/intellij/util/SonarLintUtils.java

```diff
@@ -231,7 +233,21 @@ public static void configureProxy(String host, ServerConfiguration.Builder build
       builder.proxy(proxy);
 
       if (httpConfigurable.PROXY_AUTHENTICATION) {
-        builder.proxyCredentials(httpConfigurable.PROXY_LOGIN, httpConfigurable.getPlainProxyPassword());
+        Object proxyLogin = null;
+        // Different way to fetch login based on runtime version (SLI-95)
+        try {
+          Field proxyLoginField = HttpConfigurable.class.getField("PROXY_LOGIN");
+          proxyLogin =  proxyLoginField.get(httpConfigurable);
+        } catch (NoSuchFieldException ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)